mne/inverse_sparse/_gamma_map.py

```diff
@@ -277,8 +277,10 @@ def gamma_map(evoked, forward, noise_cov, alpha, loose="auto", depth=0.8,
 
     # Reapply weights to have correct unit
     n_dip_per_pos = 1 if is_fixed_orient(forward) else 3
+    X_ori = X.copy()
     X = _reapply_source_weighting(X, source_weighting,
                                   active_set, n_dip_per_pos)
+    X_ori /= source_weighting[active_set][:, None]
```

Comment:
this looks bogus. _reapply_source_weighting does `X *= source_weighting[active_set][:, None]``
so I doubt it's doing the right thing but it's just a guess